server/src/v2/helpers/cipher.js

```diff
@@ -24,26 +25,37 @@ export function encrypt(body, passphrase, method = 'aes') {
   if (!methods.includes(method))
     return false
 
-  if (method === 'none')
+  if ( method === 'publickey' ) {
+    return cryptoDefault.publicEncrypt(
+      {
+        key: passphrase,
+        oaepHash: "sha256",
+      },
+      Buffer.from(body)
```

Comment:
We don't need this chain of if/else. Please keep it the way it was before

E.g.,
```
If method === 'publickey'
   return

if method === 'none'
   return

everything else
```

Reply:
Good point; I merged your if/return statement post-writing my bit.

Reply:
Fixed in 14e0cf8